app/src/main/java/com/github/steroidteam/todolist/view/MapsActivity.java

```diff
@@ -154,6 +153,20 @@ public void onMapReady(GoogleMap map) {
         getDeviceLocation();
     }
 
+    @Override
+    public void onBackPressed() {
+        Intent returnIntent = new Intent();
+        if (marker != null) {
+            returnIntent.putExtra("location", marker.getPosition());
```

Comment:
same here for the extra. So you know if you don't have to change every file if you have an attribute in the MapsActivity or NoteActivity